spring-cloud-azure-autoconfigure/pom.xml

```diff
@@ -15,6 +15,8 @@
 
     <properties>
         <azure.applicationinsights.version>2.0.1</azure.applicationinsights.version>
+        <!-- Instrumentation Key will be filled by maven args -Dai.instrumentationKey=xxxx for release -->
+        <ai.instrumentationKey>${place-holder}</ai.instrumentationKey>
```

Comment:
Maybe this is not needed. Just inject property into application.property file.

Reply:
will remove this.